src/features/offer/components/OfferBody/OfferBody.analytics.native.test.tsx

```diff
@@ -19,7 +22,63 @@ const canOpenURLSpy = jest.spyOn(Linking, 'canOpenURL')
 
 jest.mock('features/auth/context/AuthContext')
 
+const mockUseOffer = jest.fn().mockImplementation(() => ({
+  data: offerResponseSnap,
+}))
```

Comment:
Ce serait plus simple de faire :

```suggestion
const mockUseOffer = jest.fn().mockReturnValue({
  data: offerResponseSnap,
})
```